src/reducers/ActionType.ts

```diff
@@ -120,6 +120,8 @@ export enum ActionType {
     SetSortInProgress = "SetSortInProgress",
     ResetViewState = "ResetViewState",
     ClearViewProfiles = "ClearViewProfiles",
+    SetParentCategories = "SetParentCategories",
```

Comment:
Kannst du hier auch noch das prefix dafür schreiben, also "[View Profile]". Hilft in der Zurordnung in den devtools. Nur ne Kleinigkeit